servers/rendering/renderer_rd/storage_rd/render_scene_data_rd.cpp

```diff
@@ -76,6 +76,8 @@ void RenderSceneDataRD::update_ubo(RID p_uniform_buffer, RS::ViewportDebugDraw p
 		ubo.eye_offset[v][3] = 0.0;
 	}
 
+	RendererRD::MaterialStorage::store_transform(main_cam_transform, ubo.main_cam_inv_view_matrix);
```

Comment:
I'm matching the [above line](https://github.com/godotengine/godot/blob/master/servers/rendering/renderer_rd/storage_rd/render_scene_data_rd.cpp#L59) for storing the inverse of a `Transform3D` without calling `.affine_inverse()`, but I am a bit confused about that

Reply:
Ah, I figured it out: in the 3.x branch there is only the camera matrix which is the inverse of the view matrix here. The view matrix is world to camera transformation, whereas the camera matrix is camera to world and they are inverses of each other